packages/eslint-config/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@ni/eslint-config",
+  "version": "1.0.0",
+  "description": "NI's ESLint Shareable Config",
+  "main": "index.js",
+  "scripts": {
+    "prepublish": "npm test",
```

Comment:
I'm curious where you think we'll go with tests in this repo. Right now we're just linting the 3 lines of `index.js`, correct? Do you envision expanding the set of tests? Do you envision running them in a pipeline?

I haven't really thought about what testing looks like for a repo like this, so I'm just curious about what kinds of errors a test suite will be able to catch.

Reply:
I took a look at both [AirBnB's](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb-base/test) and [Google's](https://github.com/google/eslint-config-google) testing. AirBnB is testing requires imports and react - neither of which we need. Humorously, they don't run the linter, and their files have indentation problems 🥇 . I think that says something for our strategy! Google just runs the linter as we do. They fail on warnings which is probably a good idea! Other than that, I feel like this is a lot like testing CSS. Sure, I could try to validate that all the things look correct—or not! 

The current strategy is actually just to validate that ESLint accepts and executes the configuration. The fact that the testing strategy is the linting strategy is an accidental, not-so-accidental approach. It will allow us test specific cases if we need to do so.

Reply:
@rajsite will add the test to the pipeline. We don't think we need to add more tests.